src/types/Modifier.ts

```diff
@@ -8,6 +8,8 @@ interface Modifier<T extends string, V> {
 interface ColorGenericModifier<T extends ColorModifierTypes, V> extends Modifier<T, V> {
   space: ColorSpaceTypes;
   format: string;
+  mathFractionDigits?: number;
+  precision?: number;
```

Comment:
```suggestion
  decimals?: number;
  precision?: number;
```

I think I want to think a bit further ahead and make these a bit more brief in naming.
We'll map it to the appropriate values in sd-transforms if needed, but I think decimals / precision are the best names for these.

Reply:
Just be aware that `mathFractionDigits` currently define how precise the calculations are done, and `precision` right now sets how many decimals the color values have in out put files.

Reply:
Let's discuss this one on Slack maybe, I wanna see if we can actually bring this to a single property by making some (imo sensible) assumptions.

Reply:
The tricky part is that for ColorJS.io precision 3 means a number like 89,4%, so ColorJS.ioprecision 3 turns out to be a color like rgb(44.3% 59.5% 89.9%). So if you reuse decimals on the ColorJS.io precision, this is misleading, because for a color like rgb(44.325% 59.545% 89.992%)  you would need a color precision of 5.
I mean we could just always add 2 to the decimal setting!?

Reply:
For example, let's assume I have a mix color calculation, that with precision 4 is #c2c5d0 , for precision 2 this will be #c2c4cf  and for precision 1 this will be #ccc , that's how this works in ColorJS.io.
So this makes quite a big difference for colors, maybe having bigger impact, than for other spacing calculations?